source/guide_mailman-3.rst

```diff
@@ -0,0 +1,464 @@
+.. highlight:: console
+
+.. author:: Richard Kroegel <info@kroegel.org>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/mailman.jpg
+      :align: center
+
+#########
+Mailman 3
+#########
+
+`Mailman`_ is free software for managing electronic mail discussion and e-newsletter lists. Mailman is integrated with the web, making it easy for users to manage their accounts and for list owners to administer their lists. Mailman supports built-in archiving, automatic bounce processing, content filtering, digest delivery, spam filters, and more.
+
+`Mailman 3`_ is a complete re-write of previous versions, consisting of a suite of programs that work together:
+
+  * **Mailman Core**; the core delivery engine. This is where you are right now.
+  * **Postorius**; the web user interface for list members and administrators.
+  * **HyperKitty**; the web-based archiver
+  * **Mailman client**; the official Python bindings for talking to the Core’s REST administrative API.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Python_
+  * supervisord_
+  * Folder/File Permissions
+
+License
+=======
+
+Mailman is released under the GNU General Public License
+
+  * https://www.gnu.org/copyleft/gpl.html
+
+Prerequisites
+=============
+
+Your URL needs to be setup for web and mail:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$ uberspace mail domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Additionally, create a mailbox_ for Mailman to use to send e-mails. In this example, we are going to use ``forwarder@isabell.uber.space``.
+
+Installation
+============
+
+Get Mailman 3
+-------------
+Install Mailman 3 and its dependencies via pip.
+
+::
+
+ [isabell@stardust ~]$ pip3.6 install --user mailman postorius hyperkitty mailman-hyperkitty django whoosh
```

Comment:
`django` is already pulled in via a dependency, please remove it form the explicit list.

Reply:
Remove explicit django pull